components/datarooms/dataroom-items-list.tsx

```
@@ -51,6 +52,14 @@ import UploadZone, {
   UploadState,
 } from "@/components/upload-zone";
 
+
+import { DraggableItem } from "../documents/drag-and-drop/draggable-item";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'DraggableItem'. Consider to delete it or rename it.

'DraggableItem' is defined here:


(lint/suspicious/noRedeclare)